<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CQDlgDependencyEquation</class>
 <widget class="QDialog" name="CQDlgDependencyEquation">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>339</width>
    <height>115</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Joint Dependency Equation</string>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>91</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Joint position =</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="qqOffset">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>10</y>
     <width>81</width>
     <height>18</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>10</y>
     <width>21</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>+</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="qqCoeff">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>30</y>
     <width>81</width>
     <height>18</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>30</y>
     <width>21</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>x</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QComboBox" name="qqCombo">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>50</y>
     <width>231</width>
     <height>18</height>
    </rect>
   </property>
  </widget>
  <widget class="QDialogButtonBox" name="qqClose">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>80</y>
     <width>311</width>
     <height>32</height>
    </rect>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Close</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
